common/src/main/java/org/apache/celeborn/common/network/util/TransportConf.java

```diff
@@ -55,6 +55,18 @@ public int connectionTimeoutMs() {
     return conf.networkIoConnectionTimeoutMs(module);
   }
 
+  /** Connection heartbeat timeout in milliseconds. Default 0 secs (never). */
+  public int heartbeatTimeoutMs() {
+    return (int) JavaUtils.timeStringAsSec(
+      conf.get("rss.network.heartbeat.timeout", "0s")) * 1000;
```

Comment:
`celeborn.network.heartbeat.timeout`?

I think you need to make it the same as `networkIoConnectionTimeoutMs`

Reply:
I don't think we should make the default value to 0. According to getDanglingPingThreshold
```
  private int getDanglingPingThreshold() {
    int heartbeatTimeout = conf.heartbeatTimeoutMs() / 1000;
    int q = heartbeatTimeout / ALL_IDLE_TIMEOUT;
    int m = heartbeatTimeout % ALL_IDLE_TIMEOUT;
    return m == 0 ? q : q + 1;
  }
```
if default to zero, then threshold will be 1, which means one heartbeat timeout(5s) will cause the client timeout

Reply:
heartbeatTimeout=0
q=0
m=0
it is supposed to return 0, is there something wrong?